src/languageclient/reloadSchemas.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '../extensionVariables';
+import { ReloadShemasRequest } from './ReloadSchemasRequest';
+
+export async function reloadSchemas(): Promise<void> {
+
+    ext.outputChannel.appendLine("Reloading schemas...");
+
+    if (ext.languageServerClient) {
+        const request = new ReloadShemasRequest();
+        await ext.languageServerClient.sendRequest<ReloadShemasRequest>('schema/reloadSchemas', request);
```

Comment:
With IReloadSchemasRequest as an interface, I'd just do:

await ext.languageServerClient.sendRequest<IReloadShemasRequest>('schema/reloadSchemas', {});

Reply:
Would we even need the interface then? I wasn't sure if we needed that parameter type on the server side. I simply didn't see any examples not using any, and the documentation isn't great - so I was just following the copy/modify routine :)

Reply:
I don't believe I used an interface at all in my sample I sent you.  It's not a bad idea to have it - it will add intellisense and compile-time checking to the arg parameter, but my guess is it will work without it.

Reply:
We don't seem to need the interface. I'm deleting it.